src/utils/legacy-resolver-settings.ts

```diff
@@ -98,6 +96,18 @@ export function normalizeConfigResolvers(
   return [...map.values()]
 }
 
+export const LEGACY_NODE_RESOLVERS = [
+  'node',
+  'eslint-import-resolver-node',
+  (function () {
+    try {
+      return cjsRequire.resolve('eslint-import-resolver-node')
+    } catch {
+      // ignore
+    }
+  })(),
+].filter(Boolean)
```

Comment:
This is slow. There is only one possible falsy value here.


```suggestion
export const LEGACY_NODE_RESOLVERS = [
  'node',
  'eslint-import-resolver-node',
]

try {
  LEGACY_NODE_RESOLVERS.push(cjsRequire.resolve('eslint-import-resolver-node'))
} catch {
  // ignore
}
```

Reply:
Also, use `Set` instead of array, `Set#has` is way faster.